pkg/apis/garden/types.go

```diff
@@ -1178,6 +1178,25 @@ type KubeAPIServerConfig struct {
 	AdmissionPlugins []AdmissionPlugin
 	// AuditConfig contains configuration settings for the audit of the kube-apiserver.
 	AuditConfig *AuditConfig
+	// ServiceAccountConfig contains configuration settings for the service account handling
+	// of the kube-apiserver.
+	ServiceAccountConfig *ServiceAccountConfig
+	// APIAudiences are the identifiers of the API. The service account token authenticator will
+	// validate that tokens used against the API are bound to at least one of these audiences.
+	// If `serviceAccountConfig.issuer` is configured and this is not, this defaults to a single
+	// element list containing the issuer URL.
+	APIAudiences []string
+}
+
+// ServiceAccountConfig is the kube-apiserver configuration for service accounts.
+type ServiceAccountConfig struct {
+	// Issuer is the identifier of the service account token issuer. The issuer will assert this
+	// identifier in "iss" claim of issued tokens. This value is a string or URI.
+	Issuer *string
+	// SigningKeyFile is the path to the file that contains the current private key of the service
```

Comment:
Is should not be the path but the key file itself. Most probably it should be the reference to a secret in the same namespace as it contains a private key?